backend/clubs/models.py

```diff
@@ -1885,7 +1885,6 @@ class Ticket(models.Model):
     group_size = models.PositiveIntegerField(null=True, blank=True)
     transferable = models.BooleanField(default=True)
     attended = models.BooleanField(default=False)
-    # TODO: change to enum between All, Club, None
```

Comment:
I think you can keep this TODO haha, it's something we'll get around to in the future.